www/grpc/proto/zarb.proto

```diff
@@ -75,6 +75,7 @@ message BlockchainInfoRequest {
 
 message BlockchainInfoResponse {
   int64 height=1;
+  string stamp=2;
```

Comment:
Use `Last_block_hash` instead. 
Stamp is not a blockchain thing. Stamp is a Tx related thing.

Reply:
sure applyed